prov/efa/src/rxr/rxr_ep.c

```diff
@@ -2193,6 +2205,8 @@ void rxr_ep_inc_tx_op_counter(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry
 	 * and the RMA is a local read toward the endpoint itself
 	 */
 	peer = rxr_ep_get_peer(ep, pkt_entry->addr);
+	if (peer)
+		dlist_insert_tail(&pkt_entry->entry, &peer->outstanding_tx_pkts);
```

Comment:
Why is this the right place for this call?  And maybe document this can happen...

Reply:
Added more comments here, and renamed the function `rxr_ep_inc_tx_op_counter` to `rxr_ep_record_tx_op_submitted` because this function is called every time a TX op was submitted successfully.